server/controller/src/main/java/ai/starwhale/mlops/domain/job/ModelServingService.java

```diff
@@ -286,6 +286,8 @@ private void deploy(
         var md = modelMapper.find(model.getModelId());
 
         var envs = new HashMap<String, String>();
+        envs.put("SW_RUNTIME_PYTHON_VERSION", runtime.getVersionMetaObj().getEnvironment().getPython());
```

Comment:
as python version and sw version are mandatory， how about populate them directlly from runtime such as:
`runtime.getSwVersion()` `runtime.getPythonVersion()`

Reply:
done